src/operator/tensor/matrix_op.cc

```diff
@@ -829,8 +829,8 @@ will be the last dimension.
 
 Examples::
 
-  x = [1, 2]
-  y = [3, 4]
+  x = mx.nd.array([1, 2])
```

Comment:
please remove changes to these two lines, since mx.nd is only for Python and your example covers this.